clusters/app.ci/registry-access/cspi-qe/admin_oadp-qe.yaml

```diff
@@ -0,0 +1,24 @@
+---
```

Comment:
I didnt understand the use of this file, and why reference to oadp qe in file name

Reply:
Ah, thank you for catching this! This file is used to define permissions in the ci registry for the cspi-qe namespace. I meant to rename this file as I used that OADP version of the file as a template.